inc/common/admin-bar.php

```diff
@@ -174,13 +174,14 @@ function rocket_admin_bar( $wp_admin_bar ) {
 		/**
 		 * Purge OPCache content if OPcache is active.
 		 */
+		$opcache_enabled  = ini_get( 'opcache.enable' );
```

Comment:
[Per the PHP Manual for `ini_get()`](https://www.php.net/manual/en/function.ini-get.php#refsect1-function.ini-get-notes):
>Note: When querying boolean values
>A boolean ini value of off will be returned as an empty string or "0" while a boolean ini value of on will be returned as "1". The function can also return the literal string of INI value.

What does this mean?

For values, it returns:

- Enabled: `'1'` or `'On'` => returns `'1'` of type `string`
- Disabled:  `''`, `'0'`, or `'Off'` => returns `''` (empty string) of type `string`

Notice the type returned is a `string` and not a `boolean`.

In PHP 5 for conditional expressions, we're used to PHP type casting a `string` type into a `boolean`. However, with PHP 7, we can now be strict and declarative with our data types. For example, here we could ensure `$opcache_enabled` has a `boolean` data type.

Why? What's the point? Our variables represent data or a value. By setting the data type, we are being more _purposeful_ and _intentional_ in our code. It becomes more understandable and readable for us humans. And in some cases, it's a tiny bit more performant as PHP doesn't have to do the type casting when evaluating the value/data within a conditional expression.

Here and in each cause where we expect a boolean value:

```php
$opcache_enabled  = (bool) ini_get( 'opcache.enable' );
```



How can 

A better approach here is to `ini_get( 'opcache.enable' )`

Reply:
What if the ini setting is invalid and not what's expected? What if there's a typo such as ``10`` when it should be `'1'`? Then using `(bool)` would return `true`. Whoopsie.

How do we guard against typos or incorrect enable/disable values in the php.ini? 

We can use [`filter_var`](https://www.php.net/manual/en/function.filter-var.php) with the [`FILTER_VALIDATE_BOOLEAN` filter](https://www.php.net/manual/en/filter.filters.validate.php):

```php
$opcache_enabled = filter_var( ini_get( 'opcache.enable' ), FILTER_VALIDATE_BOOLEAN );
```
What happens here?

- Gets the setting value
- Filters it through the `FILTER_VALIDATE_BOOLEAN` filter
      - Returns `TRUE` if the setting is `'1'`, '`true'`, `'on'`, or `'yes'`
      - Else, returns 'FALSE` (for boolean and non-boolean values)